Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -4414,7 +4412,7 @@ namespace AssetProcessor
                 source.m_analysisFingerprint.append(builderFP.ToString<AZStd::string>());
             }
 
-            m_pathDependencyManager->RetryDeferredDependencies(source);
+            m_pathDependencyManager->QueueSourceForDependencyResolution(source);
```

Comment:
The second change is that the the source entry is added to the queue that gets processed later for dependency resolution.